full/src/main/resources/extended.txt

```diff
@@ -144,3 +124,20 @@ apoc.redis.zadd
 apoc.redis.zcard
 apoc.redis.zrangebyscore
 apoc.redis.zrem
+apoc.static.get
+apoc.static.list
+apoc.static.set
+apoc.systemdb.execute
+apoc.systemdb.graph
+apoc.ttl.expire
+apoc.ttl.expireIn
+apoc.uuid.install
+apoc.uuid.list
+apoc.uuid.remove
+apoc.uuid.removeAll
+apoc.data.email
```

Comment:
Note sure if it matters for when this is re-generated, but everything above this line seems to be in alphabetical order, the rest is not

Reply:
Yep, but that's the order the build generates in, we cannot do much about that, although I don't have an explanation for that, but seems to be a stable order